instructure/canvas-ios
# Course Sync Pages Interactor

This PR includes:
- container logic for downloading all kinds of content for a course
- pages download logic

---

refs: MBL-16721
affects: Student
release note: none
test plan: 
1. Have internet on 
2. Enable `offline_mode` feature flag
3. Open Course Sync menu 
4. Select Pages for download.
5. Turn off internet 
6. Open course, go to pages 
7. You should see a list of pages 

## Checklist
